### ISSUE TITLE
:rotating_light: Finding a new owner for this crate

### ISSUE THREAD

**Issue by jamwaffles** [jamwaffles/sh1106#37]
I've been burned out on maintenance of this crate for a long time now, and it's not fair to those who use it, or would like to see something better under the crate name on crates.io.

Therefore, I'd like to find a new maintainer for this crate. My only request is that I be kept in `package.authors` in `Cargo.toml` and the current Git history is preserved. Beyond that, I'll be happy to move these crates to your username, or https://github.com/rust-embedded-community/ if nobody steps up.

**Comment by trinhanhtuan** [comment]
Hi,
I have just found and used your crate few days. It seems having bug with OLED & ESP32. I posted it [here](https://github.com/embedded-graphics/embedded-graphics/issues/758).

Because I will use it for my project, I will try fixing it. By chance seeing your posting to look for a new owner for SH1106, I think I can try.

How can we move forward? Any detail requirements for this owner transferring?

Thank you,
Tuan
